templates/rhba70-full-mysql-persistent.yaml

```diff
@@ -280,7 +280,6 @@ objects:
       application: "${APPLICATION_NAME}"
     annotations:
       description: The Business Central web server's http port.
-      service.alpha.openshift.io/dependencies: '[{"name": "${APPLICATION_NAME}-mysql", "kind": "Service"}]'
```

Comment:
Curious why the removal of the dependencies (here and 3 other places)? Don't we want to make sure the mysql service is available before others that depend on it?

Reply:
@errantepiphany this was removed from the console which was its only purpose. See https://github.com/openshift/origin-web-console/issues/2319

Reply:
ok, I squashed the 7 commits

Reply:
@ruromero Okay, thanks. I thought it was an actual service dependency (not just a console UI thing), but after consulting a couple other people, that doesn't seem to be the case.